azure-resources/src/main/java/io/opentelemetry/contrib/azure/resource/IncubatingAttributes.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.azure.resource;
+
+import io.opentelemetry.api.common.AttributeKey;
+
+// copied from opentelemetry-semconv-incubating
```

Comment:
What's the motivation in copying instead of depending?

Reply:
it's a policy that library instrumentation must not depend on incubating libraries - all other contrib modules do that too